cmake/package.cmake

```diff
@@ -0,0 +1,43 @@
+if(TARGET Cryptopp::static_library)
+    return()
+endif()
+
+set(
+    CRYPTOPP_INCLUDE_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+set(
+    CRYPTOPP_STATIC_LIBRARY_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/lib"
+)
```

Comment:
in this file redundant vars are not good since it is included in a project's CMakeLists